recipes/miktex-portable/post-link.bat

```diff
@@ -0,0 +1,5 @@
+rem packages for nbconvert
+for %%x in (adjustbox booktabs collectbox fancyvrb ifoddpage mptopdf ucs url) do (
+	%PREFIX%\Library\miktex-portable\miktex\bin\mpm --install %%x  && if errorlevel 1 exit 1
+)
```

Comment:
You might want to call `mpm --update-db` first, if you're sure about installing latex packages automatically upon installation of the package.